### ISSUE TITLE
Kindly Request: Paper Update

### ISSUE THREAD

**Issue by FaceAdapter** [atfortes/Awesome-Controllable-Generation#2]
I would be extremely grateful if you could kindly consider including our latest paper, code, and Gradio in the project. We recently made them open source just a few days ago and would be honored to have them included:

Face Adapter for Pre-Trained Diffusion Models with Fine-Grained ID and Attribute Control
https://github.com/FaceAdapter/Face-Adapter
https://arxiv.org/pdf/2405.12970

**Comment by atfortes** [comment]
Of course! Feel free to make a PR with the addition of the paper to the list

Thanks!

**Comment by atfortes** [comment]
@FaceAdapter
